README.md

```diff
@@ -1 +1,156 @@
-# To-do_list
\ No newline at end of file
+<div align="center">
+  <h2><b>To do List </b></h2>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 To do List <a name="about-project"></a>
+
+**To do List** is a personal project where I implement knowledge aquired from ES6 and Webpack lessons to develop a To do list web app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+     <li><a href="https://reactjs.org/">CSS</a></li>
+     <li><a href="https://reactjs.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add task:** This project has a feature to enable user add a task to the to do list
+- **Remove task:** This feature allows the user to remove task from the to do list.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [OPTIONAl] In order to keep our documentation professional, please make sure the Readme does not have any broken links present, currently, your Live demo link is broken and giving a 404 error. please fix this by either including the correct live demo link or disabling this link :+1: